### DISCUSSION TITLE
Please implement a way to silence catching the word "stuck"

### DISCUSSION THREAD

**Discussion by malonnnn** [GriefPrevention/GriefPrevention#1922] [Ideas]
Whenever the word "stuck" is mentioned in chat, it always pops up with a grief prevention message.

There is no way to disable this in the config.

Additionally, it also happens on worlds where grief prevention claims in the config.

Thank you!

**Comment by Jikoo** [comment]
Pretty sure it can be disabled by replacing the message for it in your messages.yml with an empty string (not deleting it - it'll regenerate!)

**Reply by malonnnn** [reply]
It would still be nice to disable, because the regex still runs for nothing then

**Reply by Jikoo** [reply]
Seems like a microoptimization to me - you're talking about adding a toggle for a single regex pass made off the main thread.
